tests/unit/test_mysql.py

```diff
@@ -463,6 +463,21 @@ def test_is_instance_configured_for_innodb(self, _run_mysqlsh_script):
         )
         self.assertFalse(is_instance_configured)
 
+    @patch("charms.mysql.v0.mysql.MySQLBase._run_mysqlsh_script")
+    def test_drop_group_replicatoin_metadata_schema(self, _run_mysqlsh_script):
```

Comment:
small typo

```suggestion
    def test_drop_group_replication_metadata_schema(self, _run_mysqlsh_script):
```

Reply:
addressed in 0718c94afe58b870cb573c0b59d72b3dd37212b8